src/platform_impl/apple/appkit/window_delegate.rs

```diff
@@ -426,9 +457,16 @@ declare_class!(
 
         /// Invoked when the dragging operation is cancelled
         #[method(draggingExited:)]
-        fn dragging_exited(&self, _sender: Option<&NSObject>) {
+        fn dragging_exited(&self, info: Option<&ProtocolObject<dyn NSDraggingInfo>>) {
```

Comment:
For @madsmtm: is this the right way to represent a sender param that may be null? [The Swift docs note that for this specific callback, this parameter is nullable](https://developer.apple.com/documentation/appkit/nsdraggingdestination/draggingexited(_:)).

(As a side node, kinda alarming how the Objective-C docs just don't mention its nullability at all, as far as I can tell. Any idea what's going on there?)

Reply:
Yes, this is correct, see also [the docs in `objc2-app-kit`](https://docs.rs/objc2-app-kit/0.3.0/objc2_app_kit/trait.NSDraggingDestination.html#method.draggingExited). In the future, I plan to make it required to have the same signature as in `objc2-app-kit`, but haven't gotten around to it yet.

The docs on Apple's website don't mention it, probably to be less verbose, but the header does contain a nullability attribute:
```objc
// AppKit.framework/Headers/NSDragging.h
- (void)draggingExited:(nullable id <NSDraggingInfo>)sender;
```

Reply:
Nit: Please keep the name as `sender`.